docs/contribute.rst

```diff
@@ -16,8 +16,8 @@ Get in touch
 - On IRC find us at `#readthedocs`_.
 
 .. _StackOverFlow: https://stackoverflow.com/questions/tagged/read-the-docs
-.. _on GitHub: https://github.com/rtfd/readthedocs.org
-.. _Gitter: https://gitter.im/rtfd/readthedocs.org
+.. _on GitHub: https://github.com/readthedocs/readthedocs.org
+.. _Gitter: https://gitter.im/readthedocs/readthedocs.org
```

Comment:
I don't think `Gitter` should be changed.

Reply:
https://gitter.im/readthedocs/readthedocs.org yeah, looks like gitter doesn't get updated :/